src/ontodia/diagram/paperArea.tsx

```diff
@@ -94,7 +95,25 @@ interface PointerMoveState {
     restoreGeometry: RestoreGeometry;
 }
 
+interface ViewportState {
+    readonly center: Vector; // paper coords
```

Comment:
Please move `// paper coords` comment as JSDoc to `center` (something like `Center of the viewport in paper coordinates.`)